src/main/java/org/onebusaway/alexa/AuthedSpeechlet.java

```diff
@@ -92,15 +92,24 @@ public void onSessionEnded(final SessionEndedRequest request,
     private SpeechletResponse tellArrivals() {
         ObaArrivalInfoResponse response = obaUserClient.getArrivalsAndDeparturesForStop(userData.getStopId());
         ObaArrivalInfo[] arrivals = response.getArrivalInfo();
-        StringBuilder sb = new StringBuilder();
-        for (ObaArrivalInfo obaArrival: arrivals) {
-            log.info("Arrival: " + obaArrival);
-            ArrivalInfo arrival = new ArrivalInfo(obaArrival, response.getCurrentTime());
-            sb.append(arrival.getLongDescription() + " -- "); //with pause between sentences
+        if (arrivals.length == 0) {
+            PlainTextOutputSpeech out = new PlainTextOutputSpeech();
+            out.setText("There are no upcoming arrivals at your stop.");
+            // Is there a way to say "in the next N minutes"?  I didn't see
```

Comment:
There should be another constructor with two arguments for getting the arrival times via the `ArrivalInfoRequest` - the second argument is `minutesAfter`, which is how far out OBA looks for arrival times for that request.  We should start using that constructor, and pass in a value, which would be the same value repeated here.  The value of `minutesAfter` is a UX question, as some stops only have arrivals every 30-60 min (which results in only a record or two returned), while some transit centers have a dozen or more arrivals in the next 30 min.  So we would read back a significantly larger number of arrivals for a transit center.  I'd suggest either 35 or 65 as the default.  35 is the current server-side default, which is what we currently get if we don't provide the REST API parameter.  We recently bumped that to 65 in OBA Android.  35 probably seems reasonable for an initial value in OBA Alexa.